t/utils/utils.go

```diff
@@ -195,7 +197,7 @@ func GetNetworkID() int {
 	case fmt.Sprintf("%d", params.StatusChainNetworkID), "statuschain":
 		return params.StatusChainNetworkID
 	}
-
+	log.Debug("invalid network id, fallback to statuschain", "networkid", *networkSelected)
```

Comment:
The same here, add a separate error message for the main net, so someone who misuses the code at knows that that's intentional.